docs/README.md

```diff
@@ -0,0 +1,13 @@
+# Better Tax Handling Plugin
+
+<img src="images/better-tax-handling.png" alt="Better Tax Handling WordPress Plugin">
+
+## Introduction
+
+**Better Tax Handling** is a plugin for WooCommerce stores that simplifies the complex part of taxation for B2B and B2C selling. It is developed by [**WooCart**](https://woocart.com).
```

Comment:
**Better Tax Handling** is a plugin for WooCommerce stores that simplifies the complex part of taxation for B2B and B2C selling. It is developed with EU countries in mind, but it can help handle taxes for any countries selling to the EU.

Developed by [**WooCart**](https://woocart.com).